### ISSUE TITLE
Narrator announces dropdown aria-label along with PivotItem aria-label when the focus is on PivotItem

### ISSUE THREAD

**Issue by nicgoel** [microsoft/fluentui#12532]
### Environment Information

- **Package version(s)**: 7.84.1 and above
- **Browser and OS versions**: Edge Chromium

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/nichit/pen/ExNRyBx
- Go to the codepen link above.
- Turn on Narrator and activate the "Recent" tab.
- Listen to the Narrator announcement.

#### Actual behavior:
Narrator is announcing aria-label of both Dropdown component as well as PivotItem component

#### Expected behavior:
Narrator should annouce only the aria-label of only PivotItem component 

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) http://project.microsoft.com/

**Comment by aneeshack4** [comment]
@nicgoel I modified your codepen to have something more substantial in the first pivot too to make sure this is a Pivot issue and not a Dropdown: https://codepen.io/aneeshack4/pen/oNXRQRr?editors=1010
I confirm repro that Narrator is reading out loud each pivot item's name before any component within:
![image](https://user-images.githubusercontent.com/4161791/78416377-3b387780-75dd-11ea-878a-8e6bd9469708.png)
I took a look at Pivot.base.tsx and PivotItem.tsx and didn't see anything about their aria label hookups that would cause this. I also inspected the DOM and played around with removed the aria-labelled by relationship between the pivot tab and the button or dropdown inside and it seemed to get rid of the extra announcement but I don't see any ARIA spec violation wrt aria-labelledby in Pivot or PivotItem. @jdhuntington might know more.
